web_app/api/user.py

```diff
@@ -298,3 +299,38 @@ async def allow_notification(
         raise HTTPException(status_code=404, detail=str(e))
     except Exception as e:
         raise HTTPException(status_code=500, detail="Internal server error")
+
+
+@router.post(
+    "/api/withdraw-all",
+    tags=["User Operations"],
+    summary="Withdraw all tokens from user's contract",
+    response_description="Status of withdrawal operations",
+)
+async def withdraw_all(wallet_id: str) -> dict:
+    """
+    Withdraws all supported tokens from the user's contract.
+
+    :param wallet_id: The wallet ID of the user.
+    :return: detail: "Successfully initiated withdrawals for all tokens"
+    """
+    try:
```

Comment:
why do you need this try execept?

so you have one for  ` await CLIENT.withdraw_all(contract_address)` which is one